docs/blog/2019-07-03-themes-are-stable/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Introducing Gatsby Themes
```

Comment:
"(Formally) Introducing Gatsby Themes"?

"Announcing the Stable Release of Gatsby Themes"?

We already have a post with this title, so differentiating it in some way would be good. https://www.gatsbyjs.org/blog/2018-11-11-introducing-gatsby-themes/

Reply:
augh, I missed that when I copy/pasta'd the post structure from that original post, thanks.